importer/AtDe.py

```diff
@@ -0,0 +1,177 @@
+            else:
+                # single match - add
+                match_item = [types[x]["items"]
+                              for x in types if x.lower() == type_match][0][0]
+                self.substitute_statement("is", match_item)
+        else:
+            # no matches - report
+            self.add_to_report("type", self.name, "is")
+
+    def update_labels(self):
+        """
+        Set the labels.
+
+        The data does not contain any markup,
+        so we're adding them as is.
+        """
+        self.add_label("de", self.name)
+
+    def set_descriptions(self):
+        """
+        Set descriptions in several languages.
+
+        Base format: $type in $country
+        If specific type in German exists, substitute for $type.
+        If municipality name exists, insert in all languages
+        """
+        type_match = self.get_type_keyword()
+        municipality_name = self.get_municipality_name()
+        base_desc_german_english = "{} in {}"
+        base_desc_swedish = "{} i {}"
+        place_german = "Österreich"
+        place_english = "Austria"
+        place_swedish = "Österike"
+        type_german = "Denkmalgeschütztes Objekt"
+        type_english = "cultural property"
+        type_swedish = "kulturarv"
+
+        if isinstance(type_match, str):
+            type_german = type_match.title()
+        if municipality_name:
+            place_german = municipality_name
+            place_swedish = "{}, {}".format(municipality_name, place_swedish)
+            place_english = "{}, {}".format(municipality_name, place_english)
+        desc_german = base_desc_german_english.format(
+            type_german, place_german)
+        desc_english = base_desc_german_english.format(
+            type_english, place_english)
+        desc_swedish = base_desc_swedish.format(type_swedish, place_swedish)
+
+        self.add_description("de", desc_german)
+        self.add_description("en", desc_english)
+        self.add_description("sv", desc_swedish)
+
+    def set_street_address(self):
+        """
+        Set the street address.
+
+        Most of these look good. Patterns for invalid ones:
+        * 251, gegenüber
+          Filtering by "starts with digit" and "contains comma",
+          as we can't rely only on "starts with digit" because
+          10. Oktober-Straße 18 is legit.
+        * zwischen Entensteinstraße 36 u. 38
+          Filtering by "zwischen".
+        * 76
+          Filtering by containing only digits.
+        """
+        bad_address = None
+        if self.has_non_empty_attribute("adresse"):
+            if (utils.first_char_is_number(self.adresse) and
+                    "," in self.adresse):
+                bad_address = self.adresse
+            elif "zwischen" in self.adresse:
+                bad_address = self.adresse
+            elif self.adresse.isdigit():
+                bad_address = self.adresse
+            else:
+                address = self.adresse
```

Comment:
Would it be clearer to have this as the `else` for `if bad_address`? Separating the finding-bad-ones logic from the report-or-add logic.

Also not sure `address = self.adresse` is needed since you only use it on the line below.